gtest/src/log.rs

```diff
@@ -2,13 +2,16 @@ use crate::program::ProgramIdWrapper;
 use codec::{Codec, Encode};
 use gear_core::{message::Message, program::ProgramId};
 use std::fmt::Debug;
+use gear_core::message::{MessageId, Payload};
+use crate::System;
 
 #[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
 pub struct CoreLog {
     source: ProgramId,
```

Comment:
```suggestion
    id: MessageId,
    source: ProgramId,
```

Following other-core code styling